src/commands/init.js

```diff
@@ -279,6 +279,7 @@ const loadAbiFromBlockScout = async (ABI, network, address) =>
 const getEtherscanLikeAPIUrl = (network) => {
   switch(network){
     case "mainnet": return `https://api.etherscan.io/api`;
+    case "arbitrum-one": return `https://api.arbiscan.com/api`;
```

Comment:
Actually I think it's `.io` and not `.com`.

```
https://api.arbiscan.io/api?module=transaction&action=gettxreceiptstatus&txhash=0xe9975702518c79caf81d5da65dea689dcac701fcdd063f848d4f03c85392fd00&apikey=YourApiKeyToken
```

Example found in: https://arbiscan.io/apis#transactions